tests/test_analyses.py

```diff
@@ -5,25 +5,32 @@
 from flask import url_for
 
 
-def test_roc(client):
-    with (Path(__file__).parent / 'roc.csv').open('rb') as f:
+def upload_test_dataset(client, file, changes=None):
```

Comment:
Refactor test dataset uploading into a separate function

Reply:
This would be cleaner as a parametrized fixture.  https://docs.pytest.org/en/stable/fixture.html#parametrizing-fixtures